saleor/order/calculations.py

```diff
@@ -525,6 +525,60 @@ def order_line_tax_rate(
     return order_line.tax_rate
 
 
+def order_line_unit_discount(
+    order: Order,
+    order_line: OrderLine,
+    manager: PluginsManager,
+    lines: Optional[Iterable[OrderLine]] = None,
+    force_update: bool = False,
+) -> Decimal:
+    """Return the line unit discount.
+
+    It takes into account all plugins.
```

Comment:
Maybe would be worth extending the description to explicitly write, what exactly `unit_discount` means - like including vouchers, promotions etc. This is something that I would have to double-check in case of using the function.